build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'java'
+    id 'idea'
+}
+
+sourceCompatibility = 1.8
+
+group 'com.test'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation group: 'io.cucumber', name: 'cucumber-java', version: '6.9.1'
+    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.0'
+    testCompile group: 'io.cucumber', name: 'cucumber-junit', version: '6.9.1'
+    compile group: 'io.appium', name: 'java-client', version: '7.4.1'
+    compile 'io.github.cdimascio:dotenv-java:2.2.0'
+    implementation 'com.github.javafaker:javafaker:1.0.2'
+}
```

Comment:
Group these statements so impelementation, compile and testCompile lines are together.

Reply:
corrected